pkg/cmd/run.go

```diff
@@ -36,6 +36,7 @@ func addRunFlags(cmd *cobra.Command, cfg *config.MicroshiftConfig) {
 	flags.String("service-cidr", cfg.Cluster.ServiceCIDR, "The IP range in CIDR notation for services in the cluster.")
 	flags.String("service-node-port-range", cfg.Cluster.ServiceNodePortRange, "The port range to reserve for services with NodePort visibility. This must not overlap with the ephemeral port range on nodes.")
 	flags.String("base-domain", cfg.BaseDomain, "The base domain for this cluster.")
+	flags.String("kas-advertise-address", cfg.KASAdvertiseAddress, "The KAS advertise address for pods to connect.")
```

Comment:
Do we need a command line flag?

Reply:
I would like to standardize on just using the configuration file, and dropping the environment variables and command line flags. We can always add this later if we find a reason not to do that.

Reply:
Done